modules/stochastic_tools/include/state_sim/model/StateSimBase.h

```diff
@@ -0,0 +1,123 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#ifndef STATESIMBASE_H
+#define STATESIMBASE_H
+
+#include "MooseRandom.h"
+#include <string>
+#include <stdexcept>
+
+#define UNUSED(expr) do { (void)(expr); } while (0)
+
+typedef double TimespanH; //hours
+
+const TimespanH YEAR_TIME(8760);
+const TimespanH DAY_TIME(24);
+const TimespanH HOUR_TIME(1);
+const TimespanH ZERO_TIME(0);
```

Comment:
These should be members of the class, don't put these in the global namespace. Also, you should use the newer c++11 `constexpr` syntax.